README.md

```diff
@@ -0,0 +1,71 @@
+# Pomodoro project
+
+The Pomodoro Technique is a time management method based on 25-minute stretches of focused work broken by five-minute breaks. Longer breaks are taken after four consecutive work intervals. Each work interval is called a pomodoro, the Italian word for tomato.
+
+## Description
+
+The first thing the user will notice when starting the app is the timer. It starts with 25 minutes and, once the "beggin" button is clicked, it will start a countdown until it reaches zero. During this time frame the user is expected to be focused working on a task without interruptions. Once the timer hits 0, it will automatically begin a new countdown, this time from 5 minutes instead of 25. This time frame is meant to serve as a break period in which the user can unwind and relax for a short time. Once this "work-relax" cycle has been repeated 4 times, a new countdown begins. This time the relax period lasts for 15 minutes, which serves as a reward for having completed 4 work cycles. The app keeps track of the cycle count and what kind of break the user is supposed to take.
+A pause and a continue button have been implemented. This allows for more flexibility for the user, who can adapt to unplanned interruptions. For this same reason a restart button has been implemented which restarts the current time frame. 
+The user also has the option to write a list of tasks to better visualize what needs to be done and in which order.
+
+## Start developing the pomodoro app with React
+
+
+First you need to get installed [Node.js](https://nodejs.org/en) which will serve us as a package manager.
+*Recommended version: LTS*
+
+
+Then, check if you have installed Node correctly, running in a terminal the following command:
+`npm -v`
+*This command should just display the version you have currently installed.*
+
+
+Next, enter the project folder and run:
+`npm install`
+
+
+Once that has finished, run:
+`npm start`
+
+
+To see a live version of the app, go to [This link](http://localhost:3000) to view it in your browser.
+
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+
+---
+
+
+To launch the test runner in the interactive watch mode, run:
+`npm test`
+
+
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+
+For building the app for production run:
+`npm run build`
+*It correctly bundles React in production mode and optimizes the build for the best performance.*
+
+
+### Your app is ready to be deployed!
```

Comment:
This line shouldn't say ### Your app is ready to be deployed!#.
Instead, replace with **Your app is ready to be deployed!**